### ISSUE TITLE
Support for abstract in latex pdf

### ISSUE THREAD

**Issue by amichuda** [executablebooks/jupyterbook-latex#31]
Hiya,

It's me again! 

How feasible do you think it would be to allow some sort of abstract to be put in the in the latex pdf. This could either be implemented as an option in `_config.yml` or as a standalone admonition?

If the latter, I think it would be better to post this in the myst markdown repository, correct?

**Comment by AakashGfude** [comment]
Hey @amichuda, sorry not totally clear about what you mean by abstract. Do you mean `latex code`, `preamble`, `abstract` section of the page or something else?

**Comment by amichuda** [comment]
Yes, like an abstract in an academic paper for example. A short indented summary under the title.

**Comment by AakashGfude** [comment]
At present, we have support for only the `introduction` front matter, which is the master doc. (implemented in this package). Master doc is specified in the `toc` file. So, I reckon one way to handle this will be to specify it in `_toc.yml`, which @mmcky  had suggested earlier.
 
I think, it is definitely an architecture issue that needs a proper discussion from the myst community.

**Comment by amichuda** [comment]
Great, thank you for the reply. Is there an admonition I can use for now as a workaround?

**Comment by mmcky** [comment]
> Great, thank you for the reply. Is there an admonition I can use for now as a workaround?

Sorry @amichuda I have missed this thread. This may require a manual update of `_build/latex/<name>.tex` and then running `xelatex` or `pdflatex` as a manual step. You can build `latex` only using `jb build <dir> --builder=latex`

**Comment by mmcky** [comment]
It would be nice to make `sphinx-abstract` which adds an `abstract` directive perhaps for inserting into `jb-article`

Such as

````
```{abstract}
<abstract text>
```
````
